### ISSUE TITLE
"Multiple" in home page map legend

### ISSUE THREAD

**Issue by JChipault** [USGS-WiM/whispers#595]
"Multiple" is listed in legend under "Diagnosis Type" in legend on home page map but it's not a diagnosis type, it's an indication of multiple events in one county (which might actually all have the same diagnosis sometimes). And why is it in italics?

Neil bug report 270

**Comment by BlakeDraper** [comment]
@lprivette let's talk about this one on a hangout. The map symbology is trying to do a lot on this map and so it makes a clear, intuitive legend a challenge. We can improve it, but lets figure out a plan.

**Comment by lprivette** [comment]
Did my take of a fix for this on branch `595`.
![image](https://user-images.githubusercontent.com/23640741/59933456-8818dc00-9417-11e9-8680-00ef11b41d03.png)
I pulled it out of the diagnosis type and gave "Multiple Events" it's own section. Hate to make the legend longer, but I couldn't think of a way to make it fit in Event type or Diagnosis type.

**Comment by BlakeDraper** [comment]
@lprivette I think the approach is good. I will request a tweak of the language though - instead of "Collective Event" it should say "Multiple Diagnosis Types" - the header indicates the multiple event thing, and then we are telling the user that gray means more than one diagnosis type. Nevermind they could all be the same type. 

Basically there is no good answer here unless we refactor the whole symbology, which is not feasible right now.

**Comment by JChipault** [comment]
Can make the header "Multiple Events in One County" or is that too wordy? That would help explain the reason they are stacked on top of each other. Also, can we add some white space below the "Multiple Diagnosis Types" text such that this part of the legend is a bit separated from the "Number indicates event count"?

**Comment by lprivette** [comment]
How's this?
![image](https://user-images.githubusercontent.com/23640741/59935123-16429180-941b-11e9-8e6c-8987cd46debf.png)
I can put more space beneath the "Multiple Diagnosis Types" if desired.

**Comment by BlakeDraper** [comment]
@JChipault fair point. We are already indicating to the user that the location of the points is a county centroid and not the actual location. I want to say "Multiple Events in County" just to clarify the point, but that will also fail to translate to Canada or any other country w/out counties. "Multiple Events in the Same Level Two Administrative Unit" does not have a great ring to it.

**Comment by JChipault** [comment]
The field is called "County (or equivalent)" so could we do "Multiple Events in One County (or equivalent)"?... looks like maybe the parenthetical will fit without having the bold header be 3 lines deep.

**Comment by BlakeDraper** [comment]
Done in code
